DESCRIPTION

```diff
@@ -1,15 +1,15 @@
 Package: RQuantLib
 Title: R Interface to the 'QuantLib' Library
-Version: 0.4.17
-Date: 2022-10-25
+Version: 0.4.18
```

Comment:
Please reverse this.  Release versions are for releases, this ain't one.

What I have been doing for years in all my packages is to add a .0 or .1 at the end and increment that. I consider this to be the task of the package maintainer and do not do it for them / impose it on them.

Reply:
Done